### PR TITLE
chore: team tab walkthrough ui fixes

### PR DESCRIPTION
**Pull request by ice-hades**

### CODE REVIEW

File: src/screens/Team/components/TierList/components/Header/components/ActiveUsers.tsx [hunk 1]
```diff
@@ -6,32 +6,30 @@ import {referralsSelector} from '@store/modules/Referrals/selectors';
...
+  const isEnglishLocale = useIsEnglishLocale();
+  return isEnglishLocale ? (
+    <Text style={styles.title}>
+      <Text style={styles.label}>{`${t('users.active')}:`}</Text>
+      {` ${active}/${total}`}
+    </Text>
+  ) : (
+    <View style={styles.container}>
+      <ActiveUsersIcon width={rem(16)} style={styles.icon} />
+      <Text style={styles.titleNonEnglish}>{`  ${active}/${total}`}</Text>
```
Comment (ice-hector) [inline src/screens/Team/components/TierList/components/Header/components/ActiveUsers.tsx:29]: For RTL also tested?

**Review by ice-orion — DISMISSED** [review]

**Review by ice-hector — DISMISSED** [review]

File: src/screens/Team/components/TierList/components/Header/components/ActiveUsers.tsx (hunk 1 above)
Comment (ice-hades) [reply src/screens/Team/components/TierList/components/Header/components/ActiveUsers.tsx:29]: yes

File: src/screens/Team/components/TierList/components/Header/hooks/useActiveUsersWalkthrough.tsx [hunk 2]
```diff
@@ -21,9 +20,12 @@ export const useActiveUsersWalkthrough = ({
   const elementRef = useRef<View>(null);
 
   const {setWalkthroughElementData} = useSetWalkthroughElementData();
+  const [rendered, setRendered] = useState(false);
+  const [notSet, setNotSet] = useState(true);
```
Comment (ice-atlas) [inline src/screens/Team/components/TierList/components/Header/hooks/useActiveUsersWalkthrough.tsx]: `notSet, setNotSet` IMHO naming here not really the best choice. Let's use `isReady, setReady` or smth similar

File: src/screens/Team/components/TierList/components/Header/hooks/useEarningsWalkthrough.tsx [hunk 3]
```diff
@@ -22,8 +21,12 @@ export const useEarningsWalkthrough = ({
 
   const {setWalkthroughElementData} = useSetWalkthroughElementData();
 
+  const [rendered, setRendered] = useState(false);
+  const [notSet, setNotSet] = useState(true);
```
Comment (ice-atlas) [inline src/screens/Team/components/TierList/components/Header/hooks/useEarningsWalkthrough.tsx]: same here with naming

File: src/screens/Team/components/TierList/components/Header/hooks/useActiveUsersWalkthrough.tsx (hunk 2 above)
Comment (ice-hades) [reply src/screens/Team/components/TierList/components/Header/hooks/useActiveUsersWalkthrough.tsx]: it is used to track if setWalkthroughElementData called or not. IMHO naming is fine

File: src/screens/Team/components/TierList/components/Header/hooks/useEarningsWalkthrough.tsx (hunk 3 above)
Comment (ice-hades) [reply src/screens/Team/components/TierList/components/Header/hooks/useEarningsWalkthrough.tsx]: it is used to track if setWalkthroughElementData called or not. IMHO naming is fine

File: src/screens/Team/components/TierList/components/Header/hooks/useActiveUsersWalkthrough.tsx (hunk 2 above)
Comment (ice-hades) [reply src/screens/Team/components/TierList/components/Header/hooks/useActiveUsersWalkthrough.tsx]: fixed

File: src/screens/Team/components/TierList/components/Header/hooks/useEarningsWalkthrough.tsx (hunk 3 above)
Comment (ice-hades) [reply src/screens/Team/components/TierList/components/Header/hooks/useEarningsWalkthrough.tsx]: fixed

**Review by ice-atlas — APPROVED** [review]

**Review by ice-orion — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**